install/install.go

```diff
@@ -443,7 +443,10 @@ func resolveDst(dst string) string {
 func cleanOld(state *client.GooGetState, pi goolib.PackageInfo, insFiles map[string]string, dbOnly bool) {
 	st, err := state.GetPackageState(pi)
 	if err != nil {
-		return
+		if strings.Contains(err.Error(), "no match found for package") {
```

Comment:
Add a TODO here to use error wrapping (may require a bit of work throughout googet):
https://blog.golang.org/go1.13-errors

Reply:
done

Reply:
Testing: 
```
PS C:\Users\test> googet remove fhgqwads
ERROR: 2020/02/21 13:40:24.075262 googet_remove.go:64: Package "fhgqwads" not installed, cannot remove.
PS C:\Users\test> googet remove osqueryi
The following packages will be removed:
  osqueryi.x86_64 2020.01.15@289929166
Do you wish to remove osqueryi and all dependencies? (y/N): y
Removing osqueryi and all dependencies...
Removal of osqueryi completed
PS C:\Users\test> googet remove osqueryi
ERROR: 2020/02/21 13:41:34.192645 googet_remove.go:64: Package "osqueryi" not installed, cannot remove.
PS C:\Users\test> googet remove this isn't real
>> '
ERROR: 2020/02/21 13:42:01.203281 googet_remove.go:64: Package "this" not installed, cannot remove.
ERROR: 2020/02/21 13:42:01.207459 googet_remove.go:64: Package "isnt real\n" not installed, cannot remove.
```